lib/src/client.dart

```diff
@@ -1157,4 +1157,77 @@ class TwitchClient {
     );
     return TwitchTokenRefresh.fromJson(data);
   }
+
+  /// Create a clip
+  ///
+  /// `broadcasterId` required
+  /// `hasDelay` required
+  ///
+  /// documentation : https://dev.twitch.tv/docs/api/reference/#create-clip
+  Future<TwitchCreateClipResponse> createClip({required String broadcasterId, required bool hasDelay}) async {
+    final data = await twitchHttpClient.postCall<Map<String, dynamic>>(
+      ['clips'],
+      <String, Object>{},
+      queryParameters: {'broadcaster_id': broadcasterId, 'has_delay': hasDelay},
+    );
+    return TwitchCreateClipResponse.fromJson(data);
+  }
+
+  /// Get a clip
+  ///
+  /// `broadcasterId` exclusive Required An ID that identifies the broadcaster
+  /// `gameId` exclusive Required An ID that identifies the game whose clips you want to get
+  /// `ids` exclusive Required An ID that identifies the clip to get
+  /// `startedAt` format RFC3339
+  /// `endedAt`   format RFC3339
+  /// `first` The maximum number of clips to return per page in the response.
+  /// `before` The cursor used to get the previous page of results
+  /// `after` The cursor used to get the next page of results
+  /// `isFeatured`
+  ///
+  /// documentation : https://dev.twitch.tv/docs/api/reference/#get-clips
+  Future<TwitchClipResponse> getClips({
+    required String broadcasterId,
+    required String gameId,
+    required List<String> ids,
```

Comment:
If I'm referring to the documentation, only one of `broadcasterId`, `gameId` or `ids` is actually required by the request. Make all three parameters optional, and update the assertion check to ensure that **only one** of those isn't `null`.